frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx

```diff
@@ -147,15 +149,21 @@ const BareMetalHostDetails: React.FC<BareMetalHostDetailsProps> = ({
               <>
                 <dt>Power Status</dt>
                 <dd>
-                  <StatusIconAndText
-                    title={powerStatus}
-                    icon={<BareMetalHostPowerStatusIcon powerStatus={powerStatus} />}
-                  />
-                  {isHostScheduledForRestart(host) && (
-                    <StatusIconAndText
-                      title={HOST_SCHEDULED_FOR_RESTART}
-                      icon={<RebootingIcon />}
-                    />
+                  {!hasPowerManagement(host) ? (
+                    <SecondaryStatus status={HOST_NO_POWER_MGMT} />
```

Comment:
Shouldn't this be `StatusIconAndText`? SecondaryStatus is used as an additional info for Status

Reply:
I wanted to show `No power management` as grayed out text which `SecondaryStatus` provides. I could change that to `StatusIconAndText` and add `className`. But I dont really see that as better solution.